go/ephemeral/device_ek.go

```diff
@@ -45,99 +48,101 @@ func postNewDeviceEK(ctx context.Context, g *libkb.GlobalContext, sig string) er
 }
 
 func getServerMaxDeviceEK(ctx context.Context, g *libkb.GlobalContext) (maxGeneration keybase1.EkGeneration, err error) {
-	deviceEKs, err := GetOwnActiveDeviceEKMetadata(ctx, g)
+	deviceEKs, err := GetActiveDeviceEKMetadata(ctx, g)
 	if err != nil {
 		return maxGeneration, err
 	}
-
-	for _, deviceEK := range deviceEKs {
-		if deviceEK.Generation > maxGeneration {
-			maxGeneration = deviceEK.Generation
-		}
```

Comment:
Oh this was a bug, right? My bad.

Reply:
i think i introduced it i was mixed up on what we were returning originally